test/common/testlib.py

```diff
@@ -833,15 +833,15 @@ def check_issue(self, trace):
             if fnmatch.fnmatchcase(trace, match) or fnmatch.fnmatchcase(trace, match.replace("[", "?")):
                 number = n
         if not number:
-            return False
+            return 0
```

Comment:
Nitpick: Returning `None` would be a bit clearer (we don't have an issue #0, though).